packages/logging/src/servicies/logger/logger.service.ts

```diff
@@ -34,7 +34,7 @@ export class LoggerService implements Logger {
       ...inputOptions
     };
 
-    if (typeof window === "undefined" && config.STD_OUT_LOG_FORMAT === "pretty") {
+    if (typeof window !== "undefined" && config.STD_OUT_LOG_FORMAT === "pretty") {
```

Comment:
I'm pretty sure this should stay as is. Why'd you change this?
```suggestion
    if (typeof window === "undefined" && config.STD_OUT_LOG_FORMAT === "pretty") {
```

Reply:
sorry, i just confused

Reply:
![image](https://github.com/user-attachments/assets/0b51b997-1585-44cf-8536-9b84d015bb03)
i am getting this error when i use LoggerService 
would you check this? @ygrishajev

Reply:
how about using `this.pino.child()` instead of `this.pino.setBindings()` ?
```js
setContext(context: string) {
    this.pino.child({ context });

    return this;
  }
```
like this @ygrishajev

Reply:
and i think making two logger may good choice but it's worst case  because next.js is running on both of ssr and csr